### ISSUE TITLE
keepTracking is set to true or false by default?

### ISSUE THREAD

**Issue by soapraj** [kidh0/jquery.idle#13]
The documentation says it is set to true but from code it looks like it defaults to false. Can you please check and update one of these?

keepTracking  [ default = true ] # if you want to keep tracking user idleness, set it to true

I guess the documentation needs to be fixed.

**Comment by kidh0** [comment]
Hey @soapraj, I saw that issue yesterday and wasn't able to fix it until today. I updated the default value to true. https://github.com/kidh0/jquery.idle/commit/db0b3be7733223be6fc9969e582b1db75cbc2362

**Comment by soapraj** [comment]
Hey,

Thanks for fixing this. I have couple of more questions

1. Isnt default value being false better? It would use setTimeouts instead of setIntervals. Its better to avoid setIntervals right?

2. **if you don't want to keep tracking user even after the first time**  - I dont understand this. 

With keepTracking flag we only decide whether we use setInterval or setTimeout. Whenever the user trigger any of these events 'mousemove keypress mousedown touchstart' we call resetTimout. So the library keeps tracking all the time and not only first time. 

Please clarify this. Thanks.

**Comment by kidh0** [comment]
1 - No, this plugin was created as a way to continuously check if user is idle. The keepTracking option was add as a workaround to check it only once. The plugin uses setTimeouts (if you want to avoid to keep tracking) and setInterval (if you want do keep tracking). 
2 - I made a total mess trying to doc the plugin. Sorry about that. I fix the doc to be more clear (and less dumb). 

I took the time to check the latests contributions and I figured that it broke the plugin. The keepTracking option wasn't working as intended (check just once). I removed all the setIntervals calls and now is using only setTimout (setInterval make no sense if you want to track just once and was making the code more complex than it should). 

Thanks for the questions, it make me think about somethings that I didn't think before
